prow/cmd/deck/job_history.go

```diff
@@ -379,11 +373,24 @@ func getJobHistory(url *url.URL, config *config.Config, gcsClient *storage.Clien
 	bch := make(chan buildData)
```

Comment:
I think this could probably do with a comment regarding the intent
`// concurrently fetch data for all of the builds to be shown`